src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,10 @@
-const booleanToWord = (boolean) => {};
+const booleanToWord = (boolean) => {
```

Comment:
You could write this into one line by taking advantage of the ternary operator.
````
const booleanToWord = (boolean) => (boolean ? "Yes" : "No");
`````
In this case you are passing in boolean, if boolean has a truthy value (everything is truthy apart from: false, 0, -0, 0n, "", null, undefined, and NaN.) it will return Yes otherwise it will return No